src/api/java/mekanism/api/MekanismAPI.java

```diff
@@ -135,12 +147,20 @@ private static <T> ResourceKey<Registry<MapCodec<? extends T>>> codecRegistryKey
     public static final Registry<MapCodec<? extends RobitSkin>> ROBIT_SKIN_SERIALIZER_REGISTRY = new RegistryBuilder<>(ROBIT_SKIN_SERIALIZER_REGISTRY_NAME)
           .create();
 
-    //TODO: Potentially define this with DeferredHolder for purposes of fully defining them outside of the API
     /**
      * Empty Chemical instance.
      *
      * @since 10.7.0
+     *
+     * @deprecated Use {@link #EMPTY_CHEMICAL_HOLDER}
      */
+    @Deprecated(forRemoval = true, since = "10.7.11")//TODO - 1.22: Inline creation instead of creating the instance in the API
```

Comment:
while it will be breaking, should we just remove this? chances are slim that someone uses it

Reply:
It seems to be [used](https://waifu.neoforged.net/d/aec29a5ymel1cc/waifu-classes?orgId=1&from=2025-03-08T12:07:11.627Z&to=2025-03-08T18:07:11.627Z&timezone=browser&var-version=1.21.1-neoforge&var-class=mekanism%2Fapi%2FMekanismAPI&var-method=.%2A&var-field=%5EEMPTY_CHEMICAL$&viewPanel=panel-20) by a few mods